Core/DataObjects/ProfitTrailerData.cs

```diff
@@ -304,21 +304,12 @@ private void BuildSellLogData(dynamic rawSellLogData)
         sellLogData.AverageBuyPrice = rsld.avgPrice;
         sellLogData.TotalCost = sellLogData.SoldAmount * sellLogData.AverageBuyPrice;
 
-        // check if sale was a short position
```

Comment:
Did this not work, are you just reverting it to previous?

Reply:
Please read the comments above, as well as those in our dev channel.  I didn't fully roll-back your gui changes, I just moved them a bit.

As for the short position, the code I added way back when, just didn't work so I just removed it.

I also found some problems with the code to identify forumulas I am fixing now before this is merged.

Reply:
Ok